packages/components/app/styles/components/tabs.scss

```diff
@@ -9,14 +9,14 @@ $indicator-height: 3px;
 .hds-tabs {}
 
 // Sub-components:
-// TODO: MOVE!
+// TODO: MOVE?
 .hds-tab-list {
   padding-bottom: round(calc($indicator-height / 2));
   // overflow-x: auto;
   // -webkit-overflow-scrolling: touch;
   // white-space: nowrap; // move
 
-  // Un-nest after moving
+  // Un-nest?
```

Comment:
Yes, we tend to make the CSS as flat (lowest specificity) as possible. 

_Given the naming convention we're using, there's zero risk of conflicts or overrides (unless intentional)._

Reply:
Thanks, I will unnest. 

(Comment/Explanation: I like to keep css relatively flat but I like nesting BEM child element classes and modifiers so it's easier to see the structure and relationships. To still keep the CSS flat and avoid repeating the parent selector, I normally use partial class names for the child elements like `&__child`.)

Reply:
Personally, I tend to prefer findability of classes in the code base. Some resources on this, that explain better what I mean:
https://csswizardry.com/2017/02/code-smells-in-css-revisited/#string-concatenation-for-classes
https://bradfrost.com/blog/post/sass-selectors-to-nest-or-not-to-nest/
https://bradfrost.com/blog/post/css-architecture-for-design-systems/

Reply:
I haven't normally needed to search for classes in that way but anyway I unnested those styles.